netbox/circuits/graphql/filters.py

```diff
@@ -19,66 +45,165 @@
 
 
 @strawberry_django.filter(models.CircuitTermination, lookups=True)
-@autotype_decorator(filtersets.CircuitTerminationFilterSet)
-class CircuitTerminationFilter(BaseFilterMixin):
-    pass
+class CircuitTerminationFilter(
+    BaseObjectTypeFilterMixin,
+    CustomFieldsFilterMixin,
+    TagsFilterMixin,
+    ChangeLogFilterMixin,
+    CabledObjectModelFilterMixin,
+):
+    circuit: Annotated['CircuitFilter', strawberry.lazy('circuits.graphql.filters')] | None = (
+        strawberry_django.filter_field()
+    )
+    term_side: Annotated['CircuitTerminationSideEnum', strawberry.lazy('circuits.graphql.enums')] | None = (
+        strawberry_django.filter_field()
+    )
+    termination_type: Annotated['ContentTypeFilter', strawberry.lazy('core.graphql.filters')] | None = (
+        strawberry_django.filter_field()
+    )
+    termination_id: ID | None = strawberry_django.filter_field()
+    port_speed: Annotated['IntegerLookup', strawberry.lazy('netbox.graphql.filter_lookups')] | None = (
+        strawberry_django.filter_field()
+    )
+    upstream_speed: Annotated['IntegerLookup', strawberry.lazy('netbox.graphql.filter_lookups')] | None = (
+        strawberry_django.filter_field()
+    )
+    xconnect_id: FilterLookup[str] | None = strawberry_django.filter_field()
```

Comment:
@jeremypng do you know whether it's necessary to define these filters explicitly? Setting them to `auto` seems to yield the same schema, but I wanted to check if you had run into any problems with this approach.
```suggestion
    xconnect_id: strawberry.auto
```

Reply:
If it can successfully infer the filter type from the field type it will work. From what I saw, this generally works for model fields that are physical database fields but not computed fields.

Reply:
That makes sense, thanks!